spec/index.bs

```diff
@@ -8040,19 +8040,21 @@ dictionary GPURenderPipelineDescriptor
                 1. Let |layout| be a new [$default pipeline layout$] for |pipeline| if
                     |descriptor|.{{GPUPipelineDescriptorBase/layout}} is {{GPUAutoLayoutMode/"auto"}},
                     and |descriptor|.{{GPUPipelineDescriptorBase/layout}} otherwise.
-                1. If any of the following conditions are unsatisfied:
-                    [$generate a validation error$], [$invalidate$] |pipeline|, and stop.
```

Comment:
shoot, I messed up and accidentally removed this validation error. Thankfully I caught it in a merge conflict.